card/mod/05_standard/set/type/user.rb

```diff
@@ -46,12 +46,14 @@
   # flexibility and security when configuring initial setups
 end
 
-event :setup_first_user, before: :process_subcards, on: :create, when: proc{ |c| Card::Env.params[:setup] } do
-  subcards['signup alert email+*to'] = name
-  subcards['+*roles'] = { content: Card[:administrator].name }
-
-  email, password = subcards.delete('+*account+*email'), subcards.delete('+*account+*password')
-  subcards['+*account'] = { '+*email'=>email, '+*password'=>password }
+event :setup_first_user,
+      before: :process_subcards, on: :create,
+      when: proc{ |c| Card::Env.params[:setup] } do
```

Comment:
Unused block argument - `c`. You can omit the argument if you don't care about it.<br>Space missing to the left of {.